bundles/CoreBundle/Migrations/Version20210505093841.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Exception;
+use Pimcore\Bundle\EcommerceFrameworkBundle\CartManager\CartItem;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210505093841 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'WebDAV locks with SQLite instead of file-based';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if(!file_exists(PIMCORE_SYSTEM_TEMP_DIRECTORY.'/webdav-locks.sqlite')) {
```

Comment:
Is it right to create this once via migration inside a temp directory? What if the temp directory is somehow cleared and the migration already ran? Does that mean that from there on the lock-file (`webdav-locks.dat`) will be used?

Reply:
Ah you are right. But I actually do not want to check on every request if the database tables have been set up... I will have look how to solve this differently.